cfgov/unprocessed/css/owning-a-home/module/bootstrap-tooltips.less

```diff
@@ -0,0 +1,22 @@
+// Opacity
+.opacity(@opacity) {
+  opacity: @opacity;
+  // IE8 filter
+  @opacity-ie: (@opacity * 100);
+  filter: ~"alpha(opacity=@{opacity-ie})";
```

Comment:
Removed in https://github.com/cfpb/cfgov-refresh/pull/3165/commits/adea54f8ecb4f983b08882aaf842c2e812e402bf.